apps/frontend/styles/404.module.css

```diff
@@ -0,0 +1,30 @@
+.papiez {
+    text-align: center;
+}
+.one {
+    font-size: 75px;
+}
+.txt2 {
+    font-size: 25px;
+}
+.mainref {
+    font-size: 20px;
+    border-bottom: 1px solid rgb(204, 200, 200);
+}
+.about {
+    font-size: 20px;
+    border-bottom: 1px solid rgb(204, 200, 200);
+}
+.news {
+    font-size: 20px;
```

Comment:
it's probably better idea to use `rem` instead of `px` for font size

Reply:
i will change it tomorrow